### PR TITLE
Add package building scripts

### PR DESCRIPTION
**Pull request by caleridas**
Sample build command:

dpkg-buildpackage -us -uc -nc -b --host-arch arm64

https://auterion.atlassian.net/browse/DIS-456

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```
Comment (QuentinVecchio) [inline debian/copyright:1]: Should we not use the LICENSE from mavlink repo instead?

**Review by QuentinVecchio — APPROVED** [review]
LGTM, thanks!

File: debian/copyright (hunk 1 above)
Comment (caleridas) [reply debian/copyright:1]: Indeed, I failed to fix the license in this file -- fixed now.

**Review by QuentinVecchio — APPROVED** [review]
LGTM

**Pull request merged**